queries.sql

```diff
@@ -0,0 +1,10 @@
+/*Queries that provide answers to the questions from all projects.*/
+
+SELECT * FROM animals WHERE name like '%mon';
+SELECT name FROM animals WHERE date_of_birth >= '2016-01-01' AND date_of_birth < '2019-12-31';
+SELECT name FROM animals WHERE neutered = 't' AND escape_attempts < 3;
+SELECT date_of_birth FROM animals WHERE name = 'Pikachu' OR name = 'Agumon';
```

Comment:
- [ ] Great work, since you updated the query, kindly update the screenshot. Remember that it is a requirement that your screenshots show the result of accurate data retrieved from the database. 👍

Reply:
Thank you for the feedback. It is updated according to requirements.